app/views/provider/admin/messages/outbox/index.html.slim

```diff
@@ -1,39 +1,49 @@
 - content_for :page_header_title, 'Sent Messages'
 
 - if @messages.blank?
-  p= t('provider.admin.messages.no_messages')
+  div class="pf-c-empty-state"
+    div class="pf-c-empty-state__content"
+      i class="fas fa-rocket pf-c-empty-state__icon" aria-hidden="true"
```

Comment:
Suggesting https://fontawesome.com/v4/icon/paper-plane-o

(no "outbox" icon it seems, GMail uses a paper plan for "Sent", so inspired by that)

Reply:
Perfect 👍🏻